service/http.proxy/src/test/scripts/org/kaazing/gateway/service/http/proxy/http.proxy.path.rpt

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright 2007-2015, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+connect http://localhost:8110/acceptConfigPath/example/jsp/num/file.html
+connected
+
+write method "GET"
+write parameter "name" "value"
+write version "HTTP/1.1"
+
+write header "User-Agent" "curl/7.37.1"
+write header "Host" "localhost:8110"
+write header "Accept" "*/*"
+write close
+
+read status "200" "OK"
+read version "HTTP/1.1"
+read header "Content-Length" "17"
+read header "Content-Type" "text/html;charset=ISO-8859-1"
+#read closed
+
+close
+closed
+
+
+accept http://localhost:8080/connectConfigPath/example/jsp/num/file.html?name=value
+accepted
+connected
+
+read method "GET"
+read header "Via" "1.1 kaazing"
+read header "User-Agent" "curl/7.37.1"
+read header "Host" "localhost:8080"
+read header "Forwarded" "for=127.0.0.1"
+read header "Connection" "close"
+read header "Accept" "*/*"
+read closed
+
+write status "200" "OK"
+write version "HTTP/1.1"
+
+write header "Content-Type" "text/html;charset=ISO-8859-1"
+write header "Connection" "close"
+write header content-length
```

Comment:
You want to write those 17 bytes here as the client is expecting Content-Length: 17